packages/teuchos/comm/src/Teuchos_StackedTimer.hpp

```diff
@@ -384,6 +397,13 @@ class StackedTimer
       */
      void report(std::ostream &os);
 
+    /**
+     * Return pointer to the BaseTimer corresponding to a given string
+     * @param name input string to search for
+     * @return pointer to BaseTimer (NULL if none found)
```

Comment:
Is this the same method as the one below?  The method implementation below throws if it finds `NULL`.

Reply:
Thanks @mhoemmen ! The automerge pushed it so I will make these changes in another pr tomorrow. Just need to correct the doxygen.

Reply:
The doxygen comment about returning null is correct. There are two implementations of findBaseTimer - the top level StackeTimer method that does throw an exception and the internal LevelTimer method that returns nullptr. The StackedTimer will throw if you don't find the timer, but as the we search through each sublevel of the timer tree, we don't want to throw but just return that it was not found in this particular subtree.